wire-ios-request-strategy/Sources/E2EIdentity/EnrollE2eICertificateUseCase.swift

```diff
@@ -81,6 +81,9 @@ public final class EnrollE2eICertificateUseCase: EnrollE2eICertificateUseCaseInt
         let finalizeResponse = try await enrollment.finalize(location: orderResponse.location, prevNonce: orderResponse.acmeResponse.nonce)
         let certificateRequest = try await enrollment.certificateRequest(location: finalizeResponse.location, prevNonce: finalizeResponse.acmeResponse.nonce)
 
+        // TODO: Pass the certificate chain
```

Comment:
When can the TODO be resolved?

Reply:
@KaterinaWire we should get the certificate chain in an upcoming ticket right? Do we have a ticket number I could add to the todo?

Reply:
You can pass `certificateRequest.response` converted to a string.
I also created [this ticket](https://wearezeta.atlassian.net/browse/WPB-6039) as a reminder to verify enrollment flow after the CC bump.

Reply:
covered here https://github.com/wireapp/wire-ios/pull/793/commits/eb85a1647dcc5132a9c92e895d52b01ae272ce28